### PR TITLE
Sktay add hometown

### PR DESCRIPTION
**Pull request by sktay**

### CODE REVIEW

File: sktay-sanjose.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # San Jose, CA
 
 ### Places to eat
+- Dia de Pesca
+- Sushi Kazoo
 
 ### Touristy things to do
+- Tech Museum
+- Winchester Mystery House
```
Comment (madepu81) [inline sktay-sanjose.md]: Can you please place some weblinks? :)

**Comment by sktay** [comment]
I forgot to add a comment to my pull request. So doing it now.
:baby_chick: :computer: 
Any suggestions for San Jose?

File: sktay-sanjose.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# San Jose, CA
+
+### Places to eat
+- Dia de Pesca
+- Sushi Kazoo
+
+### Touristy things to do
```
Comment (anjith05) [inline sktay-sanjose.md]: How about any nice bars to try?

**Comment by Shimelis-Dadi** [comment]
@sktay this looks good to me :

**Comment by madepu81** [comment]
Thanks @sktay

**Comment by madepu81** [comment]
Please go ahead merge to master

**Pull request merged**